docs/node/workers.md

```diff
@@ -6,53 +6,26 @@ sidebar_label: Workers
 
 ## What is a Worker?
 
-A Worker is an object that connects to the Temporal Server and executes Workflows and [Activities](/docs/node/activities) from [Task Queues](/docs/node/task-queues).
+A Worker is an object that connects to the Temporal Server and executes [Workflows](/docs/node/workflows) and [Activities](/docs/node/activities).
 Workers are run on user-controlled hosts.
-You can use the `@temporalio/worker` package's `Worker` class to create and run as many Workers as your use case demands, across any number of hosts.
+You can use the `@temporalio/worker` package's [`Worker`](https://nodejs.temporal.io/api/classes/worker.Worker) class to create and run as many Workers as your use case demands, across any number of hosts.
 
 Workers poll [Task Queues](/docs/node/task-queues) for Tasks, execute chunks of code in response to those Tasks, and then communicate the results back to the Temporal Server.
 
 As a developer, running Workers is a fairly simple procedure because the Node SDK handles all of the communication between the Worker and the Temporal Server behind the scenes.
 
 ## How to start a Worker
 
-To start a Worker, you need to pass the following two options to the `Worker.create()` function:
+To start a Worker, you need to pass the following two required options to the `Worker.create()` function:
 
 1. The `workDir`. The Node SDK will automatically register:
 
-- Activities from any `.js` files in `workDir + '/../activities'`
-- Workflows from any `.js` files in `workDir + '/../workflows'`
+- Activities exported from `workDir + '/activities.ts'` or `workDir + '/activities/index.ts'` (or `.js` when using JavaScript).
+- Workflows exported from `workDir + '/workflows/index.ts'` (Or `.js`)
 
 2. The `taskQueue` the Worker should poll.
 
-Below is an example of starting a Worker that polls the Task Queue named 'tutorial'.
+Below is an example of starting a Worker that polls the Task Queue named `tutorial`.
 
-```ts
-import { Worker } from '@temporalio/worker';
-
-main().catch((err) => {
-  console.log(err);
-  process.exit(1);
-});
-
-async function main() {
-  const worker = await Worker.create({
-    workDir: __dirname,
-    taskQueue: 'test',
-  });
-  await worker.run();
```

Comment:
maybe we should harmonize the creation of Worker, Activity, Workflows so that users dont have to memorize different verbs.

Reply:
Activities are a special case as mentioned above.
I'm open to discussing the creation of workflows / child workflows / external workflows.

I would keep `Worker.create` though, we need an async factory for instantiation.